### ISSUE TITLE
Custom code-runners don't always load on initial dev preview

### ISSUE THREAD

**Issue by ChristopherWMM** [slidevjs/slidev#1830]
**Describe the bug**

Sometimes, the initial run of the `dev` preview does not load custom `code-runner` implementations.

**Minimal reproduction**

1. Execute `npm run dev`
2. Go to preview
3. Attempt to manually run / auto-run a non-standard monaco runner
4. See `Error: Runner for language "<lang>" not found`

See [slidev-code-runner](https://github.com/ChristopherWMM/slidev-code-runner) for an example with a custom Python implementation in `./setup/code-runners.ts`

**Environment**

- Slidev version: v0.49.26 (using `npm create slidev@latest`)
- Browser: Chrome
- OS: Windows 10

**Notes**

This issue only occurs sometimes on the initial load of the `dev` preview, any subsequent loads after making edits to the project are fine. The built version of the project also appears to be fine.

**Comment by KermanX** [comment]
Are there two Slidev servers running at the same time, and on the same port?

**Comment by ChristopherWMM** [comment]
> Are there two Slidev servers running at the same time, and on the same port?

Yes! Your question made me realize that I had the in-browser tab that pops up when starting the dev server AND the VSCode Slidev extension preview active at the same time. During my testing yesterday, I did not have the preview tab opened in VSCode and it worked fine. I just tried it again with the VSCode preview window open in my editor and one of the presentation instances (looks like the first one to load) encounters the `Error: Runner for language "<lang>" not found` error.
